pkg/apis/cloud/v1beta1/backend.go

```diff
@@ -38,6 +38,12 @@ type Backend struct {
 
 	// VolumeClaim is an optional persistent volume claim to use for persistence
 	VolumeClaim *corev1.PersistentVolumeClaim `json:"volumeClaim,omitempty"`
+
+	// ProbePort probe port
+	ProbePort int32 `json:"probeport,omitempty"`
```

Comment:
I was thinking the same thing. We need to provide livenessProbe and readinessProbe to avoid changing the backend image for readinessProbe command. For example, redis-cli is the good command that can be used for readinessProbe for redis server and creating /tm/atomix-ready is good enough for redis proxy server.